lib/encoder/include/Pololu37D.h

```diff
@@ -22,8 +22,20 @@ class Pololu37D final : public Encoder {
   bool reset() override;
 
```

Comment:
Recommend putting the encoder link (https://www.pololu.com/product/2821) in a comment at the top of the header file, as a reference for people reading your library

Reply:
done